lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -95,8 +95,8 @@ private function build($data) {
       'drupal_id' => $data->uid,
       'northstar_id' => dosomething_helpers_isset($data->field_northstar_id_value, 'id'),
       'first_name' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->field_first_name), 'first_name'),
-      'last_initial' => dosomething_helpers_isset($user->field_last_name[LANGUAGE_NONE][0]['value'], 'last_initial'),
-      'photo' => dosomething_helpers_isset($user->photo, 'photo'),
+      'last_initial' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->field_last_name), 'last_initial'),
+      'photo' => dosomething_helpers_isset(dosomething_helpers_extract_field_data($user->photo), 'photo'),
```

Comment:
The `dosomething_helpers_extract_field_data` [should check if the field is set](https://github.com/DoSomething/phoenix/blob/b280f55934058d7b8a4d599acf877be2cfae880a/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L161-L162) so I think this could be simplified:

```php
'photo' => dosomething_helpers_extract_field_data($user->photo),
```